src/App/App.scss

```diff
@@ -1042,11 +1042,13 @@ button.App-button {
 .rc-slider-tooltip-inner {
   box-shadow: none;
   padding: 0.465rem 0.8rem;
-  background: var(--color-slate-600);
+
+  background: rgb(49, 54, 85) !important;
```

Comment:
why not from library, if its one time usage -- ok

Reply:
one time